src/main/java/be/syntra/devshop/DevshopFront/models/dto/RegisterUserDto.java

```diff
@@ -9,44 +9,44 @@
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.Pattern;
 
+
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 @Builder
 public class RegisterUserDto {
 
-    @Email
-    @NotBlank
+    @Email(message = "This is not a valid email")
+    @NotBlank(message = "Please enter an email")
     private String userName;
 
-    @NotBlank
+    @NotBlank(message = "Fill out a password, just to be safe")
     private String password;
 
-    @NotBlank
+    @NotBlank(message = "Confirm password")
     private String confirmedPassword;
 
-    @NotBlank
+    @NotBlank(message = "You have a first name right?")
```

Comment:
see previous comment, preferable: "This is a required field", or something in the likes